### DISCUSSION TITLE
Not show triggers

### DISCUSSION THREAD

**Discussion by Yari27** [pawelsalawa/sqlitestudio#4718] [Q&A] [answered]
### Details
The app does not show triggers.

### Steps to reproduce
Creat trigger.

### Operating system
Windows 10

### SQLiteStudio version
3.3.3

**Comment by pawelsalawa** [comment, marked as answer]
Are you saying it's not visible here?
![image](https://user-images.githubusercontent.com/481846/224147950-84f49123-6366-4edb-b3ee-6093dbbe09cd.png)

**Comment by Yari27** [comment]
OK now I see. Sorry my mistake.
